en/SUMMARY.md

```diff
@@ -119,9 +119,11 @@
     * [Pixhack v3](flight_controller/pixhack_v3.md)
     * [AUAV-X2 (Discontinued)](flight_controller/auav_x2.md) 
   * [Snapdragon Flight](flight_controller/snapdragon_flight.md)
-    * [Camera App and Optical Flow](flight_controller/snapdragon_flight_camera.md)
-    * [Snapdragon Advanced](flight_controller/snapdragon_flight_advanced.md)
-      * [Accessing I/O Data](flight_controller/snapdragon_flight_accessing_io_data.md)
+    * [Hardware Setup Example](flight_controller/snapdragon_flight_hardware_example_setup.md)
+    * [Offboard Software Installation](flight_controller/snapdragon_flight_offboard_software.md)
```

Comment:
I would call this something like "Developer Environment". The reason being that offboard usually implies "companion computer" when you're talking about a flight system.

Reply:
Good point, I like Developer Environment better as well. Just a bit weird since it's in the user guide, and thus might lead users to think that those steps are not necessary for them. (But they are!) Suggestions?

Reply:
Indeed. We have a similar problem for Crazyflie, Bebop, and [MindPx](https://hamishwillee.gitbooks.io/ham_px4userguide/content/v/pr-snappy-doc/en/flight_controller/mindpx.html) which I have not (yet) adequately dealt with (in that we explain how to build, but not why you need to). 

Suggest in the very top level  [introduction](https://hamishwillee.gitbooks.io/ham_px4userguide/content/v/pr-snappy-doc/en/flight_controller/snapdragon_flight.html) under the board image add the note:

> **Note** Unlike some other boards Snapdragon Flight is not supplied as pre-built Firmware that is automatically updated using QGroundControl. Instead you will need to set up a developer environment and build/install the firmware yourself. The sub-documents linked from the sidebar show you how. 

I like this because it also explains the sub document. It might be improved in a "marketing sense" by giving the reason for "unlike some other boards" - e.g.  Snapdragon Flight is an extensible reference platform, so unlike other boards it is not supplied as ...

You might add a similar note at the top of the developer env doc. 

Do you think that makes sense?